tabs/src/components/sample/TabConstants.jsx

```diff
@@ -26,3 +27,8 @@ export const graphVersions = {
     v1: "v1.0",
     beta: "beta"
 };
+//TO DO: Support multi languages. Attached to ADO ticket #38594
+export const TEAMS_CHANNEL_ID = "Teams Channel ID: ";
+export const CHAT = "Chat";
+export const CHAT_ID = "Chat ID: ";
+export const NO_CONNECTED_RESOURCES = "No connected resources. Please connect to a Teams or Chats instance";
```

Comment:
These aren't used anywhere anymore, we use the translation copy now. You can just remove this code
```suggestion
```

Reply:
8f0777c6e01c05aa75d67e28752e47afce3b10e0